docker/README.md

```diff
@@ -0,0 +1,63 @@
+# Build a Docker image of Dell EMC OpenManage Ansible Modules
+Build a docker image for Dell EMC OpenManage Ansible Modules by using the Docker Engine and the docker file for OpenManage Ansible Modules. Download the docker file from [here](./Dockerfile).
+
+For more information about the Docker Engine and how to install the Docker Engine, see https://docs.docker.com/engine/.
+
+## How to build the Docker image
+To build the docker image, do the following:
+1. Download and install Docker Engine in your system.
+1. Download and save the OpenManage Ansible modules docker file from [here](./Dockerfile). Ensure that this docker file is downloaded on the same system where the docker engine is installed.
+1. Run the following Command:
+    ```bash
+    docker build -t dellemc/openmanage-ansible-modules .
+    ```
+
+    An  image with the name _dellemc/openmanage-ansible-modules_ is created. The size of this image will be 379MB.
+
+    Note:
+    * Run the command from the same directory where the Dockerfile is downloaded. If not use the option `--file` to specify the path to Dockerfile.
+    * Make sure that the `docker.io` registry search path is enabled on the system.
+    * The OpenManage Enterprise docker file uses the official docker image [python:3-slim](https://hub.docker.com/_/python) as a base. Change this value if you want to use a different python image as a base.
```

Comment:
can we talk about 1 liner about Redhat ansible bundle as well?

Reply:
details specific to Redhat are not required, in case of build failure appropriate action which the customer has to take is mentioned in the readme.